frontend/src/components/NavBar.module.scss

```diff
@@ -49,12 +47,9 @@
   display: flex;
   align-items: center;
 
-  padding-right: 10px;
   color: $COLOR_BLACK100;
-  margin-top: 12px;
-  margin-bottom: 12px;
-  padding-left: 6px; // Areas meet at the middle
-  margin-left: -6px; // Added - margin to match the padding
+  margin: 12px 0;
+  padding: 0 6px;
```

Comment:
I'd remove the padding here (affects user and logout -icons) to follow the design better. Now there's quite a large gap between them
```suggestion
```

Reply:
agreed, removed now